src/engine/threaded_engine.h

```diff
@@ -398,7 +398,8 @@ class ThreadedEngine : public Engine {
   }
 
   int bulk_size() const override {
-    return profiler::Profiler::Get()->AggregateRunning() ? 0 :  BulkStatusStore::Get()->bulk_size;
+    const profiler::Profiler *prof = profiler::Profiler::Get();
```

Comment:
why would profiler::get return nullptr when it's not enabled?

Reply:
I didn't think it did either. When profiler isn't enabled in the build at all.
The ability to turn profiler off in the build will be removed soon